sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -848,25 +848,40 @@ ssize_t coap_opt_add_string(coap_pkt_t *pkt, uint16_t optnum, const char *string
 
 ssize_t coap_opt_add_uquery(coap_pkt_t *pdu, const char *key, const char *val)
```

Comment:
Use `pkt` rather than `pdu` here for consistency with the rest of nanocoap. I felt "pdu" was more descriptive and so used that in gcoap.

Reply:
changed to `pkt`.